packages/back-end/src/controllers/metrics.ts

```diff
@@ -271,10 +271,11 @@ export async function getMetric(
 }
 
 export async function getMetricsFromTrackedEvents(
-  req: AuthRequest<null, { datasourceId: string }>,
+  req: AuthRequest<{ schema: string }, { datasourceId: string }>,
   res: Response
 ) {
   const { org } = getOrgFromReq(req);
+  const { schema } = req.body;
```

Comment:
This comment is a bit out of place, but below where we say "We were unable to identify any metrics to generate for you automatically. You can still create metrics manually", would it also make sense to pass forward the error message?

Like below that message, say `The query we ran to try to look for metrics to create automatically returned this error: \n ${e}` or something like that?

IDK if that's too confusing, but it makes it clear what table we're looking for.

Reply:
That's a great suggestion - this should hopefully help a lot of people. Attached a screenshot of the treatment I landed on, would love any feedback if you have strong opinions.

<img width="518" alt="Screen Shot 2023-08-22 at 2 19 47 PM" src="https://github.com/growthbook/growthbook/assets/75274610/4f5beba3-9f64-497a-97c9-3bd3861c9c44">

Reply:
Yeah, I think even if it doesn't help our users debug, it'll be super helpful for us in support. lgtm.